app/controllers/projects_controller.rb

```diff
@@ -10,6 +10,7 @@ class ProjectsController < ApplicationController
   before_action :require_can_edit_projects!, only: [:edit, :update]
   before_action :set_project, only: [:show, :update, :edit, :destroy]
   before_action :set_census_params, only: [:show]
+  before_action :set_location_params, only: [:show]
```

Comment:
Would we consider making this a sub-page of the Project page?  I think we have rough plans to move the client list off the bottom already, and this page seems to be doing a TON of different stuff right now.

Reply:
I think that would be fine @eanders but what is the design pattern for that? I wasn't sure how that should look since the Project dashboard doesn't have tabs.

I am seeing two patterns:
* `Project Contacts.......View/Edit` as a table row (link to a new nested page)
* Buttons on the left that go to a new nested page, like "Configure Synthetic Assessments" and "Edit Project"


The first one is probably better, out of the two.. `Client Locations ..... View`

Reply:
Yep, I think the first is the right approach.  In some future iteration we can improve that page and sub-navigation overall.

Reply:
Okay lets go with that @martha - sorry for steering you towards using the existing page, should have checked with warehouse team first!